### PR TITLE
Basic Nvisits vs Area trade, and constant depth exposure time notebooks

### PR DESCRIPTION
**Pull request by rhiannonlynne**
Preliminary merge back to master with basic tool addition!

### CODE REVIEW

File: tools/ExposureTimesForConstantDepth.ipynb [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+      "r 22.4302863843 24.9637526347 2.53346625048\n",
+      "i 21.9573319547 24.4080723472 2.45074039248\n",
+      "z 20.8779608031 23.7375564311 2.85959562795\n",
+      "y 20.0817199477 22.735065822 2.65334587434\n"
+     ]
+    }
+   ],
+   "source": [
+    "for f in ('u', 'g', 'r', 'i', 'z', 'y'):\n",
+    "    simdata = opsdb.fetchMetricData(['fiveSigmaDepth'], sqlconstraint='filter=\"%s\"' % f)\n",
```
Comment (drphilmarshall) [inline tools/ExposureTimesForConstantDepth.ipynb]: @egawiser suggested focusing on the WFD visits here, I think

File: tools/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Tools for Survey Strategy White Paper Writers
+
+This folder contains some notebooks to help support people writing cadence white papers in fall 2018. 
+
+| File | Description |
+|------|-------------|
+| [BasicNvisitsAreaTrade.ipynb](BasicNvisitsAreaTrade) | Description (Lynne!) |
```
Comment (drphilmarshall) [inline tools/README.md]: Need to add the other notebook, for completeness - and can you provide one-sentence descriptions please? Thanks!

File: tools/ExposureTimesForConstantDepth.ipynb (hunk 1 above)
Comment (drphilmarshall) [reply tools/ExposureTimesForConstantDepth.ipynb]: You could issue an enhancement here, instead of making this change now, potentially.

**Review by drphilmarshall — CHANGES_REQUESTED** [review]
Just two suggestions, including one from Eric. Should be quick I think!

**Comment by rhiannonlynne** [comment]
Hey @drphilmarshall .. have another look. 

I put the outputs into the repo for the ExposureTimes notebook this time. 
I don't expect people to rerun this one (it requires an LSST stack package) and it's slow and the outputs are useful. 

I did update the README.

**Comment by drphilmarshall** [comment]
Thanks @rhiannonlynne ! Looks good. Did you mean to check in the graphical outputs in WFD+bigarea.ipynb? Happy to merge this for now, given we are at a hackathon - we can always clear out the outputs later.

**Review by drphilmarshall — APPROVED** [review]

**Pull request merged**